marathon_quiz.cpp

```diff
@@ -143,13 +146,15 @@ int main(){
             	displayQuestionsByCategory(questionHead, category);
             	break;
             case 8:
+            	displayQuestionsInPages(Question* questionHead);            	
```

Comment:
@Starfisheunyoung 
- In line 149, let's leave out the type declaration inside of the function call, it's only needes the quesionHead. 
- Please also add, the break sentence at the end of case 8.